recipes/sdl/all/conanfile.py

```diff
@@ -170,6 +173,13 @@ def package_id(self):
             del self.info.options.sdl2main
 
     def build_requirements(self):
+        if self.settings.os == "Macos" and self.settings.arch == "armv8":
```

Comment:
I think you need to do it always when cross building because older CMake versions don't handle OBJC correctly.
```suggestion
        if self.settings.os == "Macos" and tools.is_cross_building(self):
```